graphene_subscriptions/consumers.py

```diff
@@ -56,31 +47,32 @@ def websocket_receive(self, message):
             context = AttrDict(self.scope)
 
             schema = graphene_settings.SCHEMA
+            variables = payload.get("variables")
 
-            result = schema.execute(
+            result = await execute(
```

Comment:
@jaydenwindle If you have a moment to spare I'm stuck on this change here.

`graphql-core` seems to have been changed to expect a `GraphQLSchema` rather than a text schema. I've tried to parse `schema` into a `GraphQLSchema` instance but can't get it to work.

```
    def assert_schema(schema: Any) -> GraphQLSchema:
        if not is_schema(schema):
>           raise TypeError(f"Expected {inspect(schema)} to be a GraphQL schema.")
E           TypeError: Expected <Schema instance> to be a GraphQL schema.
```